pkg/loki/loki.go

```diff
@@ -74,7 +74,7 @@ type Config struct {
 	InternalServer   internalserver.Config    `yaml:"internal_server,omitempty"`
 	Distributor      distributor.Config       `yaml:"distributor,omitempty"`
 	Querier          querier.Config           `yaml:"querier,omitempty"`
-	DeleteClient     deletion.Config          `yaml:"delete_client,omitempty"`
+	CompactorClient  compactor.ClientConfig   `yaml:"compactor_client,omitempty"`
```

Comment:
Only for v2.7.0 since we released it without this PR.

Reply:
Right, but we shouldn't backport breaking changes. People upgrading from 2.7.0 to 2.7.1 would not expect a breaking change.

Can we get this fix into the 2.7 release branch without being a breaking change, e.g. not changing the config key (and CLI argument name). That should work, because the configs are compatible, right?

Reply:
Working on this right a way

Reply:
@chaudum Change this, PTAL

Reply:
Looks good!